cmd/snap-mgmt/snap-mgmt.sh.in

```diff
@@ -47,7 +47,7 @@ purge() {
         fi
     fi
 
-    units=$(systemctl list-unit-files --full | grep -vF snap.mount.service || true)
+    units=$(systemctl list-unit-files --no-legend --full | grep -vF snap.mount.service || true)
```

Comment:
Just a sanity check question: is this supported on the old systemd in Ubuntu 14.04 or Debian 9?

Reply:
Right, i'll double check this.

Reply:
Yup, works on both Debian 9 and Ubuntu 14.04